avocado_vt/vt_agent/src/avocado_vt/agent/managers/resource_backing_manager.py

```
@@ -0,0 +1,242 @@
+        """
+        Clone a new resource by the source resource's backing.
+        """
+        r, o = 0, dict()
+        try:
+            backing = self.backings[backing_id]
+            source_backing = self.backings[arguments.pop("source")]
+            pool_conn = self.pool_connections[backing.resource_pool_uuid]
+            ret = backing.clone_resource(pool_conn, source_backing, arguments)
+            if ret:
+                o["out"] = ret
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)